src/index.js

```diff
@@ -30,8 +30,11 @@ io.on('connection', (socket) => {
 
   socket.on('join', (res) => {
     console.log(`someone joined ${res.pathName}`)
-    sendEmail(res.email, res.username, res.pathName);
     socket.join(res.pathName)
+    const peopleInARoom = socket.adapter.rooms[res.pathName].length;
+    if (peopleInARoom <= 1) {
```

Comment:
I think we want to send the email if only peopleInARoom === 1.
Is there any reason you're checking for if there's no one in the room?
I know this will work tho. But for readability, what do you think?

Reply:
Alright. I'll update this logic